### ISSUE TITLE
adding new strains to datatable leads to loss of custom tracks in browser view

### ISSUE THREAD

**Issue by fanc-WU** [debugpoint136/virus-browser#20]
It seems that currently the datatable doesn't interact with track view. to replicate this: login, select 2 strains from datatable -> browser view -> add a custom bed track from https://wangftp.wustl.edu/~cfan/viralBrowser/primers/N1.bed.gz -> open datatable again  and add several more strains -> go back to browser view -> the custom track previously added is gone. It seems that every time the selection on datatable changes, the browser view is reloaded and all previous "memory" is gone.

**Comment by debugpoint136** [comment]
Please confirm if still an issue @fanc-WU

**Comment by fanc-WU** [comment]
It's still having trouble. A video is sent as a demo

**Comment by debugpoint136** [comment]
You are right. This issue still persists. I will revisit the fix and update you
